src/ActionRepeater.UI/Pages/HomePage.xaml

```diff
@@ -33,10 +33,11 @@
 
                 <AppBarSeparator Margin="5,0,5,0" />
 
+
                 <StackPanel Padding="5,0,5,0"
                             ToolTipService.Placement="Bottom"
-                            ToolTipService.ToolTip="Number of times to repeat the actions.&#x0d;&#x0a;Negtive number = Repeat infinitely">
-                    <TextBlock Text="Repeat Actions ⓘ"
+                            ToolTipService.ToolTip="Controls how often the input sequence is repeated.&#x0d;&#x0a;A negative number causes the inputs to repeat indefinitely.">
```

Comment:
what about: "Specifies the number of repetitions of the action sequence. -1: repeat indefinitely"